examples/backends/vllm/launch/agg_omni_image.sh

```
@@ -6,6 +6,7 @@ set -e
 trap 'echo Cleaning up...; kill 0' EXIT
 
 SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPT_DIR/../../../common/gpu_utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../../common/gpu_utils.sh was not specified as input (see shellcheck -x).

(SC1091)